### ISSUE TITLE
If LoadBalancer is not found then deploy.sh should recreate service with type: NodePort

### ISSUE THREAD

**Issue by rothgar** [ElasticBox/elastickube#97]
When creating elastickube in an environment that doesn't support LoadBalancer you need to edit or delete/recreate the ELASTICKUBE_SERVER_SVC to remove [type: LoadBalancer](https://github.com/ElasticBox/elastickube/blob/master/build/deploy.sh#L200) and instead expose the service as type: NodePort

After the service is deployed an admin needs to find the definition and manually run `kubectl edit --namespace kube-system svc elastickube-server` and change the type field or find the yaml definition in the script and change it there.

During your [test for if GKE](https://github.com/ElasticBox/elastickube/blob/master/build/deploy.sh#L308) you should optionally edit the service or print out the commands for a user to edit the service manually.

**Comment by albertoarias** [comment]
@rothgar Makes sense, I can see the problem. Do you get any errors when trying to deploy?

I am trying to figure out how to detect this condition.

**Comment by rothgar** [comment]
No errors, I just had to manually create the service and make it's type NodePort.

You should be able to detect if [LOADBALANCER_IP_PROPERTY](https://github.com/ElasticBox/elastickube/blob/master/build/deploy.sh#L312) is IP: then recreate the service as [type: NodePort](https://github.com/ElasticBox/elastickube/blob/master/build/deploy.sh#L200)
